### PR TITLE
Add First Quantum Qiskit Code Tutorial

### PR DESCRIPTION
**Pull request by abhishekjiitr**
### **Describe your change:**
Add First Quantum Qiskit Code Tutorial
Fixes https://github.com/TheAlgorithms/Python/issues/2923

* [x] Add an algorithm?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: quantum/q1.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import qiskit as q
+
+"""
+Build a simple bare-minimum quantum
+circuit that starts with a single qubit
+(by default in state 0), runs the experiment
+1000 times, and finally prints the total 
+count of the states finally observed.
```
Comment (cclauss) [inline quantum/q1.py:8]:
```suggestion
Build a simple bare-minimum quantum circuit that starts with a single qubit (by
default, in state 0), runs the experiment 1000 times, and finally prints the total
count of the states finally observed.
```
Lines should be wrapped at 88 characters max.

**Review by cclauss — CHANGES_REQUESTED** [review]
Thanks for doing this.  As discussed in CONTRIBUTING.md, algorithms should be Python functions or classes with type hints and doc tests.

File: quantum/q1.py (hunk 1 above)
Comment (abhishekjiitr) [reply quantum/q1.py]: Pushed requested changes in a new commit. :+1:

**Comment by abhishekjiitr** [comment]
@cclauss added new commit with requested changes

**Review by cclauss — APPROVED** [review]

**Pull request merged**